### ISSUE TITLE
Wrong node name in conflict message for repeats

### ISSUE THREAD

**Issue by tek** [tree-sitter/tree-sitter#954]
In current master at c98dc566d5bf7f73ca7717695da9c52322abb550 I get conflict messages like:

```
Unresolved conflict for symbol sequence:

  'newtype'  tyconid  •  '_varid'  …

Possible interpretations:

  1:  'newtype'  (_simpletype  tyconid  •  _forall_repeat1)
  2:  'newtype'  (constraint  tyconid  •  type_apply_repeat1)
  3:  'newtype'  (type_name  tyconid)  •  '_varid'  …

```

where the `_forall_repeat1` uses the name of an unrelated node, it should be `_simpletype_repeat1`.
Seems to be consistent for all messages I've seen.

**Comment by maxbrunsfeld** [comment]
Yeah, this happens when `forall` and `simpletype` contain an identical `repeat(...)` value. The tool needs to pick *some* arbitrary name for that repeated rule, and in this case it picked `forall`. It would be good to tweak the message somehow to make that more clear.

**Comment by tek** [comment]
ah, indeed! you want to keep the issue open?
